### PR TITLE
Update draft-wqk-opsawg-control-schedule-framework.md

### PR DESCRIPTION
**Pull request by zhangli-abcd**
Add TVR use case yang model example

### CODE REVIEW

File: draft-wqk-opsawg-control-schedule-framework.md [hunk 1]
```diff
@@ -511,11 +512,69 @@ In the context of a tidal network, If the network maintains all the devices
 up to guarantee the maximum throughput all the time, a lot of power will be
 wasted. The energy-saving methods may include the deactivation of some or all
 components of network nodes. These activities have the potential to alter
-network topology and impact data routing in a variety of ways.  Ports on
+network topology and impact data routing in a variety of ways.  Interfaces on
 network nodes can be selectively disabled or enabled based on traffic patterns,
 thereby reducing the energy consumption of nodes during periods of low network
 traffic.
 
+The network management devices need to control the state of interfaces of each node
```
Comment (QiufangMa) [inline draft-wqk-opsawg-control-schedule-framework.md:520]:
```suggestion
The controlling of the interface states of each node
```

File: draft-wqk-opsawg-control-schedule-framework.md [hunk 2]
```diff
@@ -511,11 +512,69 @@ In the context of a tidal network, If the network maintains all the devices
...
+by the ietf-tvr-node model defined in {{?I-D.ietf-tvr-schedule-yang}}.
```
Comment (QiufangMa) [inline draft-wqk-opsawg-control-schedule-framework.md:521]:
```suggestion
can be achieved by using the ietf-tvr-node YANG module defined in {{?I-D.ietf-tvr-schedule-yang}}.
```

File: draft-wqk-opsawg-control-schedule-framework.md [hunk 3]
```diff
@@ -511,11 +512,69 @@ In the context of a tidal network, If the network maintains all the devices
...
+
+The following indicates the example of a scheduling node interface that enabled at 7:00 AM and disabled at 1:00 AM, every day, from December 1, 2025 in UTC.
```
Comment (QiufangMa) [inline draft-wqk-opsawg-control-schedule-framework.md:523]:
```suggestion
The following indicates the example of a scheduling node interface that is powered on from 12 AM, December 1, 2025 to 7 AM, December 1, 2026 in UTC, and its interface named interface1 is scheduled to enable at 7:00 AM, every day, from December 1, 2025 to December 1, 2026 in UTC.
```

File: draft-wqk-opsawg-control-schedule-framework.md [hunk 4]
```diff
@@ -511,11 +512,69 @@ In the context of a tidal network, If the network maintains all the devices
...
+The JSON encoding is used only for illustration purposes.
+
+~~~~
+{
+    "ietf-tvr-node:node-schedule": [
+        {
+            "node-id": 12345678,
+            "node-power-schedule": {
+              "power-default": true
```
Comment (QiufangMa) [inline draft-wqk-opsawg-control-schedule-framework.md:532]:
```suggestion
              "power-default": false
```

File: draft-wqk-opsawg-control-schedule-framework.md [hunk 5]
```diff
@@ -511,11 +512,69 @@ In the context of a tidal network, If the network maintains all the devices
...
+              ]
+            }
+            "interface-schedule": [
+              {
+                "name": "interace1",
+                "default-available": false,
+                "default-bandwidth": 1000000000,
+                "attribute-schedule": {
+                  "schedules":[
+                    {
```
Comment (QiufangMa) [inline draft-wqk-opsawg-control-schedule-framework.md:548]: suggest to remove the first schedule instance, which is contradictory with the second one.

File: draft-wqk-opsawg-control-schedule-framework.md [hunk 6]
```diff
@@ -511,11 +512,69 @@ In the context of a tidal network, If the network maintains all the devices
...
+                    },
+                    {
+                      "schedule-id": 333333,
+                      "recurrence-first": {
+                        "utc-start-time": "2025-12-01T01:00:00Z"
+                      },
+                      "utc-until": "2026-12-01T07:00:00Z",
+                      "frequency": "ietf-schedule:daily",
+                      "interval": 1,
+                      "available":true,
```
Comment (QiufangMa) [inline draft-wqk-opsawg-control-schedule-framework.md:566]:
```suggestion
"attr-value": {
     "available":true,
}
```
needs the "attr-value" container per the ietf-tvr-node definition.

**Review by QiufangMa — CHANGES_REQUESTED** [review]

File: draft-wqk-opsawg-control-schedule-framework.md (hunk 3 above)
Comment (zhangli-abcd) [reply draft-wqk-opsawg-control-schedule-framework.md:523]: Suggest to delete the "interface" in the first line

**Review by QiufangMa — APPROVED** [review]

**Pull request merged**